modules/Kconfig.renesas_fsp

```diff
@@ -22,3 +22,19 @@ config USE_RA_FSP_I2C_IIC
 	bool
 	help
 	  Enable Renesas RA I2C IIC Master driver
+
+config USE_RA_FSP_SCE
+	bool
+	help
+	  Enable RA FSP SCE driver
```

Comment:
What is this Kconfig option for? Why doesn't `ENTROPY_RENESAS_RA_RSIP_E51A_TRNG` `select HAS_RENESAS_RA_RSIP_E51A` directly, or why is `HAS_RENESAS_RA_RSIP_E51A` guarded by `USE_RA_FSP_SCE`?

Reply:
Because the HAL interface is SCE, but it has multiple implementations for it, depending on which Security IP is located on the SoC (in that case is RSIP-E51A). I think it is better to enable the interface by *_FSP_SCE config follow the FSP interface naming but build the correct .c file by *_RSIP* config.

Reply:
I see. But why define these two Kconfig options at all? `USE_RA_FSP_SCE` is selected by `ENTROPY_RENESAS_RA_RSIP_E51A_TRNG`, but neither of `USE_RA_FSP_SCE` and `HAS_RENESAS_RA_RSIP_E51A` are actually used to do anything?

Reply:
These 2 config is using for the CMake build system in hal_renesas (the external module for Renesas HAL). You can refer to this https://github.com/zephyrproject-rtos/hal_renesas/blob/main/drivers/ra/CMakeLists.txt

Reply:
Ah, alright. Thanks for the explanation!